azure-go-aks-multicluster/go.mod

```
@@ -0,0 +1,9 @@
+module azure-go-aks-multicluster
+
+go 1.15
```

```suggestion
go 1.16
```
You might have to regen the go.sum etc. but recommend bumping to 1.16.